lib/main.dart

```diff
@@ -1,9 +1,14 @@
 import 'package:flutter/material.dart';
+import 'package:hydrated_bloc/hydrated_bloc.dart';
+import 'package:path_provider/path_provider.dart';
 
 import 'services/locator.dart';
 import 'src/app/app.dart';
 
-void main() {
+Future<void> main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+  HydratedBloc.storage = await HydratedStorage.build(
```

Comment:
Move this to `registerServices()` and add a trailing comma for better formatting if you are spanning more than one line.